src/core/client.c

```diff
@@ -170,6 +204,41 @@ xcb_rectangle_t client_clamp_rect_to_monitor(xcb_rectangle_t window_rect,
         return new_rect;
 }
 
+uint16_t client_get_active_border_width(const struct client_theme *theme,
```

Comment:
In the future these should be refactored so that we don't need two functions to get both the fields. The fields should both be scoped under the client state